### ISSUE TITLE
Model Code restriction message isn't clear

### ISSUE THREAD

**Issue by BDavydov** [3drepo/3drepo.io#2028]
### Description
User starts typing model code and chooses to use a special character like - or /. Error message comes up saying code must match the following: "/^[A-Za-z0-9]+$/"
So, if it must match then it is allowed to contain these special characters? Or is it the case of Must NOT contains the following: "/^[A-Za-z0-9]+$/" ?

### Steps to replicate
- Login to 3D Repo
- Start creating a new model in any project
- Start typing model Code and add any special character
- Error message comes up and Save button becomes grey

### Pictures/ Gifs of the error
![image](https://user-images.githubusercontent.com/19904513/81553887-c267d080-937d-11ea-939e-c9f5cea277fe.png)

**Comment by carmenfan** [comment]
@BDavydov nope, that's a regex rule that actually means alphanumeric only. We'll just change it to something more user understandable.

**Comment by BDavydov** [comment]
@carmenfan much appreciated!

**Comment by m1chalkubiak** [comment]
@carmenfan 

Did You think maybe what message we can show here?  The current one which is automatically generated by the library that we using for validation, that is why it looks so weird for custom validation regex.
But of course, we can add our own message. 

I was thinking about:
```
Model code can only contain uppercase/lowercase letters and digits (A-Z, a-z, 0-9)
```
but this seems to be too long and doesn't look good

<img width="350" alt="Zrzut ekranu 2020-08-13 o 17 10 07" src="https://user-images.githubusercontent.com/24941866/90152471-225c8180-dd88-11ea-8d92-7c438fa9d3b0.png">


so maybe simple
```
Model code can only contain alphanumeric
```
would that be enough, what do You think?

**Comment by carmenfan** [comment]
@m1chalkubiak Alphanumeric is good enough, we also have that somewhere else in the code base! :)

**Comment by m1chalkubiak** [comment]
@carmenfan 
Directly in the code, when looking for messages containing the phrase "alphanumeric", I found only this message:

```Invalid revision name; check length is between 1 and 20 and uses alphanumeric characters.```

I don't know if that is exactly what you were thinking about, so for now, I'm leaving that with the second proposed message. 

```Model code can only contain alphanumeric```

But if You feel it should be written different way, please let me know.
